test/demo/contextChannels/broadcastOnChannel.inttest.ts

```diff
@@ -34,7 +34,7 @@ describe('When attempting to broadcast on a channel object', () => {
     });
 
     test('If a valid context is provided, the broadcast() resolves successfully', async () => {
-        await expect(channel.broadcast(testContext)).resolves;
+        await channel.broadcast(testContext);
```

Comment:
TURNS OUT - `await expect(somePromise).resolves;` doesn't do what I thought it did!

The .resolves doesn't in itself return a promise, you need to call a matcher to get an actual promise, or in this case if you don't care about the result, you can just await the promise

Reply:
Huh, TIL